### ISSUE TITLE
chromedp/headless-shell: E: The repository 'http://security.debian.org/debian-security stable/updates Release' does not have a Release file.

### ISSUE THREAD

**Issue by gregtczap** [chromedp/chromedp#898]
#### What versions are you running?

```
chromedp/headless-shell:92.0.4515.131
```

#### What did you do? Include clear steps.

This issue is about chrome headless-shell. I don't know where else to post the error. Until recently my dockerfile based on headless-shell was working fine. Today it started giving an error on `apt-get update`. Could this be because the base version of ubuntu is no longer supported by canonical? I've upgraded to the latest/stable version `92.0.4515.131` and am still getting the error.

```
$ docker build -t hs-example .
Sending build context to Docker daemon  2.048kB
Step 1/4 : FROM chromedp/headless-shell:92.0.4515.131
 ---> 3c97a36fa965
Step 2/4 : ENV PATH=$PATH:/headless-shell
 ---> Using cache
 ---> 25b351a5c8c3
Step 3/4 : RUN apt-get update && 	apt-get install -y 	dumb-init 	ca-certificates 	xmlsec1 	procps
 ---> Running in 608b60e716db
Err:1 http://deb.debian.org/debian stable InRelease
  Temporary failure resolving 'deb.debian.org'
Err:2 http://security.debian.org/debian-security stable/updates InRelease
  Temporary failure resolving 'security.debian.org'
Err:3 http://deb.debian.org/debian stable-updates InRelease
  Temporary failure resolving 'deb.debian.org'
Reading package lists...
W: Failed to fetch http://deb.debian.org/debian/dists/stable/InRelease  Temporary failure resolving 'deb.debian.org'
W: Failed to fetch http://security.debian.org/debian-security/dists/stable/updates/InRelease  Temporary failure resolving 'security.debian.org'
W: Failed to fetch http://deb.debian.org/debian/dists/stable-updates/InRelease  Temporary failure resolving 'deb.debian.org'
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists...
Building dependency tree...
Reading state information...
Package ca-certificates is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Unable to locate package dumb-init
E: Package 'ca-certificates' has no installation candidate
E: Unable to locate package xmlsec1
E: Unable to locate package procps
The command '/bin/sh -c apt-get update && 	apt-get install -y 	dumb-init 	ca-certificates 	xmlsec1 	procps' returned a non-zero code: 100
```

#### What did you expect to see?

Success

#### What did you see instead?

Errors

**Comment by kenshaw** [comment]
The better place for this would be [here](https://github.com/chromedp/docker-headless-shell). I'm assuming you're trying to build a custom image here? Would it be possible to share your `Dockerfile`?. 

The issue here is likely due to an intermittent issue on the Debian security package repo/mirrors. Looking at what appears to have failed, it's your `apt-get update` in your derived Dockerfile. Remove that, and just do a `apt-get install -y` and this should start working.

**Comment by gregtczap** [comment]
@kenshaw Thanks.. I did try to file at [chromedp/docker-headless-shell](https://github.com/chromedp/docker-headless-shell) first but issues appear to be disabled there (no issues tab). I will create a shortened version of our custom Dockerfile for reproduction and post it shortly.

**Comment by gregtczap** [comment]
@kenshaw Here is a stripped down version of the Dockerfile. I'm actually unable to build it because `Sending build context to Docker daemon #GB` exceeds 10GB and continues climbing what seems like indefinitely. I've never seen that issue before, perhaps you know why. 

```docker
# Chromedp headless-shell latest/stable as of: 08/14/21
FROM chromedp/headless-shell:92.0.4515.131

ENV PATH=$PATH:/headless-shell

# ca-certificates: used for proper ssl handling
# xmlsec1: used for XML digital signatures for SAML assertions
# procps: used for enabling ps command for debugging
RUN apt-get update && \ 
	apt-get install -y \
	dumb-init \
	ca-certificates \
	xmlsec1 \
	procps

ENTRYPOINT ["dumb-init", "--"]
```

**Comment by gregtczap** [comment]
@kenshaw The issue with build context was the directory the Dockerfile was in. Moving it to it's own directory resolved the issue. I'm still able to get the build error with the above Dockerfile by attempting to build it, so I've gone ahead and updated the original issue with the exact build error for clarity.

**Comment by gregtczap** [comment]
I reduced the Dockerfile to the following:

```
FROM ubuntu:20.04
RUN apt-get update
```

And I still received the following errors. So I think we can safely say this has nothing to do with `headless-shell`. Hopefully they fix their DNS soon, thanks for the help @kenshaw .

```
$ docker build -t hs-example .
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM ubuntu:20.04
 ---> 1318b700e415
Step 2/2 : RUN apt-get update
 ---> Running in 084aaf4b38ab
Err:1 http://archive.ubuntu.com/ubuntu focal InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:2 http://security.ubuntu.com/ubuntu focal-security InRelease
  Temporary failure resolving 'security.ubuntu.com'
Err:3 http://archive.ubuntu.com/ubuntu focal-updates InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Err:4 http://archive.ubuntu.com/ubuntu focal-backports InRelease
  Temporary failure resolving 'archive.ubuntu.com'
Reading package lists...
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-updates/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://archive.ubuntu.com/ubuntu/dists/focal-backports/InRelease  Temporary failure resolving 'archive.ubuntu.com'
W: Failed to fetch http://security.ubuntu.com/ubuntu/dists/focal-security/InRelease  Temporary failure resolving 'security.ubuntu.com'
W: Some index files failed to download. They have been ignored, or old ones used instead.
Removing intermediate container 084aaf4b38ab
 ---> a1938828c09d
Successfully built a1938828c09d
Successfully tagged hs-example:latest
```

**Comment by kenshaw** [comment]
The issue here, I'm assuming, is connected with Debian's 11 release today. Apparently the `stable` dist directory, which is what the `debian:stable-slim` image has in its `/etc/apt/sources.list` file is missing. Likely, very many people are having this issue, and I'd wager that the Debian team will fix it soon, or that the Docker image will get updated soon. In the interim, you could try doing a `perl -pi -e s/stable/oldstable/ /etc/apt/sources.list` in your derived Docker image.

**Comment by kenshaw** [comment]
@gregtczap it's worth noting on your second example with Ubuntu, that has nothing to do with Ubuntu/Debian. That's a local DNS resolution failure. It's worth noting that the Ubuntu and Debian teams are not the same team. The specific issue you have with deriving an image from `chromedp/headless-shell` is a problem that any user (today) will have with Docker's standard `debian:stable-slim` image, and likely many many many other standard Docker images based on Debian. Until the Debian security repos get fixed with the missing directories, any Debian image is going to be broken. Similarly, any installation of Debian is likely going to have this issue at the moment.

**Comment by gregtczap** [comment]
@kenshaw Yep that makes perfect sense regarding all the docker image derivatives of `debian:stable-slim`, debian installs, etc. I don't believe the DNS resolution issues for me are just local however, as I originally saw similar errors being reported from the docker build on GH actions, and I'm able to `dig` the host just fine outside of the docker build:

```
$ dig archive.ubuntu.com

; <<>> DiG 9.16.1-Ubuntu <<>> archive.ubuntu.com
;; global options: +cmd
;; Got answer:
;; ->>HEADER<<- opcode: QUERY, status: NOERROR, id: 4920
;; flags: qr rd ra; QUERY: 1, ANSWER: 2, AUTHORITY: 0, ADDITIONAL: 1

;; OPT PSEUDOSECTION:
; EDNS: version: 0, flags:; udp: 4096
;; QUESTION SECTION:
;archive.ubuntu.com.		IN	A

;; ANSWER SECTION:
archive.ubuntu.com.	41	IN	A	91.189.88.142
archive.ubuntu.com.	41	IN	A	91.189.88.152

;; Query time: 20 msec
;; SERVER: 127.0.0.1#53(127.0.0.1)
;; WHEN: Sat Aug 14 17:38:33 PDT 2021
;; MSG SIZE  rcvd: 79
```

**Comment by kenshaw** [comment]
@gregtczap I've switched the `headless-shell` image over to using `buster-slim` instead of `stable-slim`. I've been looking through the associated issues / changes on the respective projects, and I don't believe the Debian team is going to fix this. Unfortunately, that means that anyone using a previous fixed base image version will need to upgrade to `latest` `beta` `dev` chromedp/headless-shell tags, as I do not have the resources to go back and republish / retag all the various `chromedp/headless-shell` builds.

**Comment by gregtczap** [comment]
@kenshaw I see... I imagined they would want to fix this issue asap but I guess the impact is limited enough that it's not a priority? Huh. Anyways good call on the switch with `headless-shell`. That's much appreciated on my end of things as our build/release cycle was blocked by this. I'm currently using `92.0.4515.131` which is also `latest` and `stable` and can confirm the fix you just pushed has addressed the issue. Thanks for responding so quickly on this.

**Comment by gregtczap** [comment]
@kenshaw I was mistaken. Still getting some debian errors from the full Dockerfile:

```
FROM chromedp/headless-shell:latest # 92.0.4515.131

RUN apt-get update && \ 
	apt-get install -y \
	ca-certificates \
	xmlsec1 \
	procps
```

Results:

```
$ docker build -t hs-test .
Sending build context to Docker daemon  2.048kB
Step 1/2 : FROM chromedp/headless-shell:latest
latest: Pulling from chromedp/headless-shell
33847f680f63: Pull complete 
740a6c7b9f3f: Pull complete 
0023fce95b0c: Pull complete 
e01d69a07df1: Pull complete 
Digest: sha256:be5a4deefa1f346be1d163a7e1506bffe8b06bb489033843346176bccd3ab059
Status: Downloaded newer image for chromedp/headless-shell:latest
 ---> b7b660ce452e
Step 2/2 : RUN apt-get update && 	apt-get install -y 	ca-certificates 	xmlsec1 	procps
 ---> Running in fc6eb02f561a
Err:1 http://security.debian.org/debian-security buster/updates InRelease
  Temporary failure resolving 'security.debian.org'
Err:2 http://deb.debian.org/debian buster InRelease
  Temporary failure resolving 'deb.debian.org'
Err:3 http://deb.debian.org/debian buster-updates InRelease
  Temporary failure resolving 'deb.debian.org'
Reading package lists...
W: Failed to fetch http://deb.debian.org/debian/dists/buster/InRelease  Temporary failure resolving 'deb.debian.org'
W: Failed to fetch http://security.debian.org/debian-security/dists/buster/updates/InRelease  Temporary failure resolving 'security.debian.org'
W: Failed to fetch http://deb.debian.org/debian/dists/buster-updates/InRelease  Temporary failure resolving 'deb.debian.org'
W: Some index files failed to download. They have been ignored, or old ones used instead.
Reading package lists...
Building dependency tree...
Reading state information...
Package ca-certificates is not available, but is referred to by another package.
This may mean that the package is missing, has been obsoleted, or
is only available from another source

E: Package 'ca-certificates' has no installation candidate
E: Unable to locate package xmlsec1
E: Unable to locate package procps
The command '/bin/sh -c apt-get update && 	apt-get install -y 	ca-certificates 	xmlsec1 	procps' returned a non-zero code: 100
```

DNS resolution appears fine:

```
$ host deb.debian.org
deb.debian.org is an alias for debian.map.fastlydns.net.
debian.map.fastlydns.net has address 151.101.54.132
debian.map.fastlydns.net has IPv6 address 2a04:4e42:d::644
```

**Comment by kenshaw** [comment]
@gregtczap this is some kind of issue with your local networking/Docker configuration. Using what you provided, this worked just fine for me:
```sh
$ cat Dockerfile 
FROM chromedp/headless-shell:latest
RUN apt-get update && \ 
	apt-get install -y \
	ca-certificates \
	xmlsec1 \
	procps
$ docker build --pull --progress=plain --no-cache --tag kenshaw/test:latest .
#1 [internal] load build definition from Dockerfile
#1 sha256:aa3d9f2aa7f43ef14e13b653061f1dc5aeb9c47e222adfa9892db86ef2a422a5
#1 transferring dockerfile: 37B done
#1 DONE 0.0s

#2 [internal] load .dockerignore
#2 sha256:69b0e259f6eb62a5481ad5da28d9783476f03aa41ba4a23b59360448f4da7bdc
#2 transferring context: 2B done
#2 DONE 0.0s

#3 [internal] load metadata for docker.io/chromedp/headless-shell:latest
#3 sha256:cd45901ae79cc6a994eab16a2b6e42e8b62d9bbd4617168aa20e10357453ecd5
#3 ...

#4 [auth] chromedp/headless-shell:pull token for registry-1.docker.io
#4 sha256:d02b37226eb7d9ecd62496091baff7c005a18bbbdcc12059255c640bb2a030bb
#4 DONE 0.0s

#3 [internal] load metadata for docker.io/chromedp/headless-shell:latest
#3 sha256:cd45901ae79cc6a994eab16a2b6e42e8b62d9bbd4617168aa20e10357453ecd5
#3 DONE 5.2s

#5 [1/2] FROM docker.io/chromedp/headless-shell:latest@sha256:be5a4deefa1f346be1d163a7e1506bffe8b06bb489033843346176bccd3ab059
#5 sha256:14d8d18b602ea9fb9c3255426585f495d3c11252cc00eeccd396f565e405c755
#5 resolve docker.io/chromedp/headless-shell:latest@sha256:be5a4deefa1f346be1d163a7e1506bffe8b06bb489033843346176bccd3ab059 done
#5 CACHED

#6 [2/2] RUN apt-get update && 	apt-get install -y 	ca-certificates 	xmlsec1 	procps
#6 sha256:d8a0e17efff22f28fa200b02378f201865f180b1b776399d58acdbeee5cc0bc3
#6 0.301 Get:1 http://deb.debian.org/debian buster InRelease [122 kB]
#6 0.317 Get:2 http://security.debian.org/debian-security buster/updates InRelease [65.4 kB]
#6 0.394 Get:3 http://deb.debian.org/debian buster-updates InRelease [51.9 kB]
#6 0.466 Get:4 http://security.debian.org/debian-security buster/updates/main amd64 Packages [301 kB]
#6 0.549 Get:5 http://deb.debian.org/debian buster/main amd64 Packages [7907 kB]
#6 2.319 Get:6 http://deb.debian.org/debian buster-updates/main amd64 Packages [15.2 kB]
#6 3.062 Fetched 8463 kB in 3s (3006 kB/s)
#6 3.062 Reading package lists...
#6 3.448 Reading package lists...
#6 3.826 Building dependency tree...
#6 3.898 Reading state information...
#6 3.959 The following additional packages will be installed:
#6 3.959   libgpm2 libicu63 libncurses6 libprocps7 libssl1.1 libxml2 libxmlsec1
#6 3.959   libxmlsec1-openssl libxslt1.1 lsb-base openssl psmisc
#6 3.959 Suggested packages:
#6 3.959   gpm
#6 4.032 The following NEW packages will be installed:
#6 4.032   ca-certificates libgpm2 libicu63 libncurses6 libprocps7 libssl1.1 libxml2
#6 4.032   libxmlsec1 libxmlsec1-openssl libxslt1.1 lsb-base openssl procps psmisc
#6 4.032   xmlsec1
#6 4.126 0 upgraded, 15 newly installed, 0 to remove and 0 not upgraded.
#6 4.126 Need to get 12.7 MB of archives.
#6 4.126 After this operation, 43.8 MB of additional disk space will be used.
#6 4.126 Get:1 http://deb.debian.org/debian buster/main amd64 libncurses6 amd64 6.1+20181013-2+deb10u2 [102 kB]
#6 4.208 Get:2 http://deb.debian.org/debian buster/main amd64 libprocps7 amd64 2:3.3.15-2 [61.7 kB]
#6 4.225 Get:3 http://deb.debian.org/debian buster/main amd64 lsb-base all 10.2019051400 [28.4 kB]
#6 4.271 Get:4 http://deb.debian.org/debian buster/main amd64 procps amd64 2:3.3.15-2 [259 kB]
#6 4.301 Get:5 http://deb.debian.org/debian buster/main amd64 libssl1.1 amd64 1.1.1d-0+deb10u6 [1539 kB]
#6 4.644 Get:6 http://deb.debian.org/debian buster/main amd64 openssl amd64 1.1.1d-0+deb10u6 [844 kB]
#6 4.794 Get:7 http://deb.debian.org/debian buster/main amd64 ca-certificates all 20200601~deb10u2 [166 kB]
#6 4.831 Get:8 http://deb.debian.org/debian buster/main amd64 libgpm2 amd64 1.20.7-5 [35.1 kB]
#6 4.839 Get:9 http://deb.debian.org/debian buster/main amd64 libicu63 amd64 63.1-6+deb10u1 [8300 kB]
#6 6.544 Get:10 http://deb.debian.org/debian buster/main amd64 libxml2 amd64 2.9.4+dfsg1-7+deb10u2 [689 kB]
#6 6.688 Get:11 http://deb.debian.org/debian buster/main amd64 libxslt1.1 amd64 1.1.32-2.2~deb10u1 [237 kB]
#6 6.798 Get:12 http://deb.debian.org/debian buster/main amd64 libxmlsec1 amd64 1.2.27-2 [148 kB]
#6 6.813 Get:13 http://deb.debian.org/debian buster/main amd64 libxmlsec1-openssl amd64 1.2.27-2 [99.5 kB]
#6 6.814 Get:14 http://deb.debian.org/debian buster/main amd64 psmisc amd64 23.2-1 [126 kB]
#6 6.815 Get:15 http://deb.debian.org/debian buster/main amd64 xmlsec1 amd64 1.2.27-2 [47.6 kB]
#6 6.900 debconf: delaying package configuration, since apt-utils is not installed
#6 6.914 Fetched 12.7 MB in 3s (4559 kB/s)
#6 6.927 Selecting previously unselected package libncurses6:amd64.
(Reading database ... 6678 files and directories currently installed.)
#6 6.929 Preparing to unpack .../00-libncurses6_6.1+20181013-2+deb10u2_amd64.deb ...
#6 6.933 Unpacking libncurses6:amd64 (6.1+20181013-2+deb10u2) ...
#6 6.956 Selecting previously unselected package libprocps7:amd64.
#6 6.957 Preparing to unpack .../01-libprocps7_2%3a3.3.15-2_amd64.deb ...
#6 6.959 Unpacking libprocps7:amd64 (2:3.3.15-2) ...
#6 6.975 Selecting previously unselected package lsb-base.
#6 6.976 Preparing to unpack .../02-lsb-base_10.2019051400_all.deb ...
#6 6.978 Unpacking lsb-base (10.2019051400) ...
#6 6.996 Selecting previously unselected package procps.
#6 6.997 Preparing to unpack .../03-procps_2%3a3.3.15-2_amd64.deb ...
#6 7.000 Unpacking procps (2:3.3.15-2) ...
#6 7.036 Selecting previously unselected package libssl1.1:amd64.
#6 7.036 Preparing to unpack .../04-libssl1.1_1.1.1d-0+deb10u6_amd64.deb ...
#6 7.038 Unpacking libssl1.1:amd64 (1.1.1d-0+deb10u6) ...
#6 7.130 Selecting previously unselected package openssl.
#6 7.131 Preparing to unpack .../05-openssl_1.1.1d-0+deb10u6_amd64.deb ...
#6 7.133 Unpacking openssl (1.1.1d-0+deb10u6) ...
#6 7.192 Selecting previously unselected package ca-certificates.
#6 7.193 Preparing to unpack .../06-ca-certificates_20200601~deb10u2_all.deb ...
#6 7.195 Unpacking ca-certificates (20200601~deb10u2) ...
#6 7.227 Selecting previously unselected package libgpm2:amd64.
#6 7.227 Preparing to unpack .../07-libgpm2_1.20.7-5_amd64.deb ...
#6 7.229 Unpacking libgpm2:amd64 (1.20.7-5) ...
#6 7.247 Selecting previously unselected package libicu63:amd64.
#6 7.248 Preparing to unpack .../08-libicu63_63.1-6+deb10u1_amd64.deb ...
#6 7.250 Unpacking libicu63:amd64 (63.1-6+deb10u1) ...
#6 7.775 Selecting previously unselected package libxml2:amd64.
#6 7.775 Preparing to unpack .../09-libxml2_2.9.4+dfsg1-7+deb10u2_amd64.deb ...
#6 7.777 Unpacking libxml2:amd64 (2.9.4+dfsg1-7+deb10u2) ...
#6 7.833 Selecting previously unselected package libxslt1.1:amd64.
#6 7.833 Preparing to unpack .../10-libxslt1.1_1.1.32-2.2~deb10u1_amd64.deb ...
#6 7.836 Unpacking libxslt1.1:amd64 (1.1.32-2.2~deb10u1) ...
#6 7.866 Selecting previously unselected package libxmlsec1:amd64.
#6 7.867 Preparing to unpack .../11-libxmlsec1_1.2.27-2_amd64.deb ...
#6 7.869 Unpacking libxmlsec1:amd64 (1.2.27-2) ...
#6 7.898 Selecting previously unselected package libxmlsec1-openssl:amd64.
#6 7.899 Preparing to unpack .../12-libxmlsec1-openssl_1.2.27-2_amd64.deb ...
#6 7.901 Unpacking libxmlsec1-openssl:amd64 (1.2.27-2) ...
#6 7.924 Selecting previously unselected package psmisc.
#6 7.925 Preparing to unpack .../13-psmisc_23.2-1_amd64.deb ...
#6 7.927 Unpacking psmisc (23.2-1) ...
#6 7.949 Selecting previously unselected package xmlsec1.
#6 7.950 Preparing to unpack .../14-xmlsec1_1.2.27-2_amd64.deb ...
#6 7.952 Unpacking xmlsec1 (1.2.27-2) ...
#6 7.977 Setting up lsb-base (10.2019051400) ...
#6 7.983 Setting up libgpm2:amd64 (1.20.7-5) ...
#6 7.989 Setting up psmisc (23.2-1) ...
#6 7.996 Setting up libssl1.1:amd64 (1.1.1d-0+deb10u6) ...
#6 8.043 debconf: unable to initialize frontend: Dialog
#6 8.043 debconf: (TERM is not set, so the dialog frontend is not usable.)
#6 8.043 debconf: falling back to frontend: Readline
#6 8.043 debconf: unable to initialize frontend: Readline
#6 8.043 debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
#6 8.043 debconf: falling back to frontend: Teletype
#6 8.059 Setting up libprocps7:amd64 (2:3.3.15-2) ...
#6 8.065 Setting up libicu63:amd64 (63.1-6+deb10u1) ...
#6 8.071 Setting up libncurses6:amd64 (6.1+20181013-2+deb10u2) ...
#6 8.077 Setting up procps (2:3.3.15-2) ...
#6 8.092 update-alternatives: using /usr/bin/w.procps to provide /usr/bin/w (w) in auto mode
#6 8.092 update-alternatives: warning: skip creation of /usr/share/man/man1/w.1.gz because associated file /usr/share/man/man1/w.procps.1.gz (of link group w) doesn't exist
#6 8.100 Setting up openssl (1.1.1d-0+deb10u6) ...
#6 8.109 Setting up libxml2:amd64 (2.9.4+dfsg1-7+deb10u2) ...
#6 8.116 Setting up ca-certificates (20200601~deb10u2) ...
#6 8.165 debconf: unable to initialize frontend: Dialog
#6 8.165 debconf: (TERM is not set, so the dialog frontend is not usable.)
#6 8.165 debconf: falling back to frontend: Readline
#6 8.165 debconf: unable to initialize frontend: Readline
#6 8.165 debconf: (Can't locate Term/ReadLine.pm in @INC (you may need to install the Term::ReadLine module) (@INC contains: /etc/perl /usr/local/lib/x86_64-linux-gnu/perl/5.28.1 /usr/local/share/perl/5.28.1 /usr/lib/x86_64-linux-gnu/perl5/5.28 /usr/share/perl5 /usr/lib/x86_64-linux-gnu/perl/5.28 /usr/share/perl/5.28 /usr/local/lib/site_perl /usr/lib/x86_64-linux-gnu/perl-base) at /usr/share/perl5/Debconf/FrontEnd/Readline.pm line 7.)
#6 8.165 debconf: falling back to frontend: Teletype
#6 8.368 Updating certificates in /etc/ssl/certs...
#6 8.664 137 added, 0 removed; done.
#6 8.685 Setting up libxslt1.1:amd64 (1.1.32-2.2~deb10u1) ...
#6 8.692 Setting up libxmlsec1:amd64 (1.2.27-2) ...
#6 8.698 Setting up libxmlsec1-openssl:amd64 (1.2.27-2) ...
#6 8.708 Setting up xmlsec1 (1.2.27-2) ...
#6 8.713 Processing triggers for libc-bin (2.28-10) ...
#6 8.723 Processing triggers for ca-certificates (20200601~deb10u2) ...
#6 8.727 Updating certificates in /etc/ssl/certs...
#6 8.965 0 added, 0 removed; done.
#6 8.965 Running hooks in /etc/ca-certificates/update.d...
#6 8.966 done.
#6 DONE 9.0s

#7 exporting to image
#7 sha256:e8c613e07b0b7ff33893b694f7759a10d42e180f2b4dc349fb57dc6b71dcab00
#7 exporting layers
#7 exporting layers 0.2s done
#7 writing image sha256:979580eaf3197246d8dd7337c632c99bcdba4933e9e55f31c24b1c685e82e86c done
#7 naming to docker.io/kenshaw/test:latest done
#7 DONE 0.2s
```

**Comment by gregtczap** [comment]
@kenshaw Okay, noted. You did point that out before as well. I'll investigate what the issue might be.
